<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsVectorTileConnectionDialog</class>
 <widget class="QDialog" name="QgsVectorTileConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vector Tiles Connection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="mGroupBox">
     <property name="title">
      <string>Connection Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="2">
       <widget class="QgsSpinBox" name="mSpinZMax">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>14</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QCheckBox" name="mCheckBoxZMax">
        <property name="text">
         <string>Max. Zoom Level</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="lblReferer">
        <property name="text">
         <string>Referer</string>
        </property>
        <property name="buddy">
         <cstring>mEditReferer</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="mEditUrl">
        <property name="toolTip">
         <string>URL of the connection, {x}, {y}, and {z} will be replaced with actual values. Use {-y} for inverted y axis.</string>
        </property>
        <property name="placeholderText">
         <string>http://example.com/{z}/{x}/{y}.pbf</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="mCheckBoxZMin">
        <property name="text">
         <string>Min. Zoom Level</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="3">
       <widget class="QGroupBox" name="mAuthGroupBox">
        <property name="title">
         <string>Authentication</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QgsAuthSettingsWidget" name="mAuthSettings" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLineEdit" name="mEditReferer">
        <property name="toolTip">
         <string>Optional custom referer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Source URL</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QgsSpinBox" name="mSpinZMin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="mEditName">
        <property name="toolTip">
         <string>Name of the new connection</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="mEditStyleUrl">
        <property name="toolTip">
         <string>If specified, the layer will be automatically styled using styling JSON from the entered URL.</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Style URL</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsAuthSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qgsauthsettingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mEditName</tabstop>
  <tabstop>mEditStyleUrl</tabstop>
  <tabstop>mEditUrl</tabstop>
  <tabstop>mSpinZMin</tabstop>
  <tabstop>mSpinZMax</tabstop>
  <tabstop>mEditReferer</tabstop>
  <tabstop>mCheckBoxZMax</tabstop>
  <tabstop>mCheckBoxZMin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsVectorTileConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsVectorTileConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
